ReactWindows/ReactNative/UIManager/UIViewOperationQueue.cs

```diff
@@ -161,7 +162,7 @@ public bool IsEmpty()
         /// Enqueues an operation to remove the root view.
         /// </summary>
         /// <param name="rootViewTag">The root view tag.</param>
-        public void EnqueueRemoveRootView(int rootViewTag)
+        public Task RemoveRootViewAsync(int rootViewTag)
```

Comment:
>RemoveRootViewAsync [](start = 20, length = 19)

I guess this is kind of misleading now, because we're still returning after the remove has been enqueued in some cases.

Reply:
I.e., the remove operation is not necessarily complete.

---
In reply to: [179480100](https://github.com/Microsoft/react-native-windows/pull/1737#discussion_r179480100) [](ancestors = 179480100)

Reply:
It looks like we're really only awaiting the potential `OnDestroy` call on the queue.

---
In reply to: [179480193](https://github.com/Microsoft/react-native-windows/pull/1737#discussion_r179480193) [](ancestors = 179480193,179480100)

Reply:
True, that's why I renamed to RemoveRootViewAsync.

---
In reply to: [179480549](https://github.com/Microsoft/react-native-windows/pull/1737#discussion_r179480549) [](ancestors = 179480549,179480193,179480100)